src/core/alloc_func.cpp

```diff
@@ -26,5 +28,5 @@ void NORETURN MallocError(size_t size)
  */
 void NORETURN ReallocError(size_t size)
 {
-	error("Out of memory. Cannot reallocate " PRINTF_SIZE " bytes", size);
+	FatalError("Out of memory. Cannot reallocate {} bytes", size);
```

Comment:
I think `PRINTF_SIZE` in stdafx.h is no longer needed.

Reply:
Since some of the other PRs got merged it's indeed not needed anymore. I've got a (local) branch that removes this one and its friends in stdafx.h together with the final bits. However, that depends on this and the other PRs.